apps/test-suite/tests/NewNotifications.js

```diff
@@ -597,6 +597,204 @@ export async function test(t) {
       });
     });
 
+    t.describe('Notification Categories', () => {
+      const vanillaButton = {
+        identifier: 'vanillaButton',
+        buttonTitle: 'Plain Option',
+        options:
+          Platform.OS === 'ios'
```

Comment:
Yeah it makes sense. The matching code further down in `setNotificationCategoriesAsync` will have to change since `jasmine` doesn't handle nested objects/objects in arrays very well. That's done now 👍